frontend/src/app/Settings/Facility/FacilityFormContainer.tsx

```diff
@@ -158,11 +158,11 @@ export const ADD_FACILITY_MUTATION = gql`
 `;
 
 interface Props {
-  facilityId: string;
   newOrg?: boolean;
 }
 
 const FacilityFormContainer: any = (props: Props) => {
+  const { facilityId } = useParams();
```

Comment:
this is what the url look like in test
https://test.simplereport.gov/app/settings/facility/7f1d9789-37b0-4a83-99fa-824403e95e90?facility=08ea10c8-a767-4d72-a60a-46dca51228bc

I see key of `facility` and not `facilityId`

Reply:
So in that case, the uuid after `/facility` is the id of the facility being edited in the form, whereas the `facility` query param refers to the currently selected facility in the nav bar. We want the id from the url params because that's the one we actually want to edit in the form.

Reply:
That makes sense, I wish it was a bit clearer which facility is which, but that could be addressed in a future refactor.